RPLCD/lcd.py

```diff
@@ -117,6 +135,8 @@ def __init__(self, cols=20, rows=4, dotsize=8, charmap='A02', auto_linebreaks=Tr
         else:
             raise ValueError('Invalid data bus mode: {}'.format(self.data_bus_mode))
 
+
+
```

Comment:
Two stray empty lines that can be removed.

Reply:
\*poof\*